README.md

```diff
@@ -5,11 +5,11 @@
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
 ## Documentation
-This repository is a part of the ShotGrid Pipeline Toolkit.
+This repository is a part of the Flow Production Tracking Toolkit Pipeline Toolkit.
```

Comment:
Well, we are not going to repeat the toolkit twice: `Flow Production Tracking Toolkit Pipeline Toolkit`

But the question is what do we do with "Pipeline Toolkit"? I see two ways:

- Forget the word pipeline: Flow Production Tracking Toolkit
- Append the work pipeline: Flow Production Tracking Toolkit Pipeline

I'd prefer option 1. But we should ask around.

Reply:
@juanburgosautoglb I got confirmation from Eric: option 1. We remove the pipeline word.